apis/python/tools/ingestor

```diff
@@ -130,18 +130,20 @@ select `relative=True`. (This is the default.)
 
 def ingest_one(input_path: str, output_path: str, ifexists: str, soma_options: tiledbsc.SOMAOptions, verbose: bool):
     # Check that the input exists.
-    if not os.path.exists(input_path):
-        # Print this neatly and exit neatly, to avoid a multi-line stack trace otherwise.
-        print(f"Input path not found: {input_path}", file=sys.stderr)
-        sys.exit(1)
+    if not (input_path.startswith('s3://') or input_path.startswith('tiledb://')):
+        if not os.path.exists(input_path):
```

Comment:
Why not use `vfs.is_dir` or `vfs.is_file`?

Reply:
As a further comment I believe we should use VFS operations for things or the TileDB object API for checking if array/group. The object API works for all backends. VFS works for everything but `tiledb://`. The current code will fail for azure/gcs/anything not covered here.